relayer/src/common/events/FillInvalidated.ts

```diff
@@ -1,7 +1,7 @@
 import type { Log } from "@ethersproject/providers";
 import { Interface } from "ethers/lib/utils";
 
-import FillManagerABI from "../../assets/abi/FillManager.json";
+import { abis } from "@/deployments";
```

Comment:
For some reason, the `@/` aliasing is not working when bundling the relayer via `pkg`. Can you use relative paths instead on all places where you used `@/`?